src/mobu/exceptions.py

```diff
@@ -281,6 +281,38 @@ class NotebookRepositoryError(MobuSlackException):
     """The repository containing notebooks to run is not valid."""
 
 
+class RepositoryConfigError(MobuSlackException):
+    """The in-repo mobu.yaml config file is invalid."""
+
+    def __init__(
+        self,
+        *,
+        err: Exception,
+        user: str,
+        repo_url: str,
+        repo_ref: str,
+        config_file: Path,
+    ) -> None:
+        super().__init__(str(err), user)
+        self.err = err
+        self.user = user
+        self.repo_url = repo_url
+        self.repo_ref = repo_ref
+        self.config_file = config_file
+
+    def to_slack(self) -> SlackMessage:
+        message = super().to_slack()
+        message.blocks += [
+            SlackTextBlock(heading="GitHub Repository", text=self.repo_url),
+            SlackTextBlock(heading="Git Ref", text=self.repo_ref),
+        ]
+        message.attachments += [
+            SlackCodeBlock(heading="Error", code=str(self.err))
```

Comment:
When reporting a different exception, prefer code like:
```suggestion
            SlackCodeBlock(heading="Error", code=f"{type(self.err).__name__}: {self.err!s}")
```
to include the type of the exception. For a lot of exceptions (`ValueError`, for instance), the exception message is often very cryptic without the exception type, and in some cases there can be no message at all.